annotations/annotation-user/pom.xml

```diff
@@ -27,6 +27,13 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        
+        <dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.16.10</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
The Java Builder Pattern does not work without additional plugins in Eclipse, Lombok provides @Builder annotation, that works fine.